### PR TITLE
Fix StyleCop issues that belong to CSharp.DocumentationRules

### PR DESCRIPTION
**Pull request by lewischeng-ms**

### CODE REVIEW

**Comment by lewischeng-ms** [comment]
Wait for #99 to check in first. Will send another PR after that.

File: src/Common.Stylecop [hunk 1]
```diff
@@ -21,6 +33,10 @@
           </RuleSettings>
         </Rule>
       </Rules>
+      <AnalyzerSettings>
+        <BooleanProperty Name="IgnorePrivates">True</BooleanProperty>
```
Comment (congysu) [inline src/Common.Stylecop]: Thanks for disabling documentation requirement to private and internal members.

File: src/Microsoft.Restier.Core/DomainBase.cs [hunk 2]
```diff
@@ -154,7 +154,7 @@ protected DomainConfiguration DomainConfiguration
         }
 
         /// <summary>
-        /// Gets a value indicating if this domain
+        /// Gets a value indicating whether this domain
```
Comment (congysu) [inline src/Microsoft.Restier.Core/DomainBase.cs]: nit: add a trailing `.`.

File: src/Microsoft.Restier.WebApi/Formatter/Serialization/DefaultODataDomainSerializerProvider.cs [hunk 3]
```diff
@@ -9,17 +9,30 @@
...
+        /// Initializes a new instance of the <see cref="DefaultODataDomainSerializerProvider" /> class.
+        /// </summary>
         public DefaultODataDomainSerializerProvider()
         {
             this.feedSerializer = new ODataDomainFeedSerializer(this);
             this.entityTypeSerializer = new ODataDomainEntityTypeSerializer(this);
         }
 
+        /// <summary>
+        /// Get the serializer for the given result type.
```
Comment (congysu) [inline src/Microsoft.Restier.WebApi/Formatter/Serialization/DefaultODataDomainSerializerProvider.cs]: nit: Gets

Comment (lewischeng-ms) [inline src/Microsoft.Restier.WebApi/Formatter/Serialization/DefaultODataDomainSerializerProvider.cs]: good.

File: src/Microsoft.Restier.Core/DomainBase.cs (hunk 2 above)
Comment (lewischeng-ms) [inline src/Microsoft.Restier.Core/DomainBase.cs]: OK

**Pull request closed without merging**